### PR TITLE
Add button to dismiss try-the-new-thing banner

### LINKED ISSUE #920: Add banner to asset detail page to try the new thing
**User story/persona**
As a user, I want to switch from the old UI to the new UI while I'm on a specific asset.

**Acceptance Criteria**

- [x] If the advertise feature flag is enabled, there should be a banner on the asset detail page that provides a link to try the new UI.
- [x] The link to try the new UI should (somehow) load the asset the user was on as the currently-opened asset.
- [x] There should be a way to dismiss the banner once and for all (until browser cookies are cleared).

Caveat: no banner when looking at completed pages.

### PR DESCRIPTION
**Pull request by rstorey**
Closes #920 

Looks like this: 
Screen Shot 2019-05-24 at 2 39 16 PM

### CODE REVIEW

File: concordia/templates/transcriptions/asset_detail.html [hunk 1]
```diff
@@ -517,6 +518,16 @@ <h5 id="captcha-modal-title" class="modal-title">Please confirm you are not a ro
 <script src="https://cdnjs.cloudflare.com/ajax/libs/openseadragon/2.4.0/openseadragon.min.js" integrity="sha256-grVJpqgDSGBx1q2llmvY+J5zU9hEHbO7UvftFY2bK1w=" crossorigin="anonymous"></script>
 <script src="{% static 'js/contribute.js' %}"></script>
 <script src="{% static 'js/asset-reservation.js' %}"></script>
+<script>
+    if(localStorage.getItem("no-interface-banner")==null) {
```
Comment (acdha) [inline concordia/templates/transcriptions/asset_detail.html]: I'm wondering whether this should be less strictly a null check:

```suggestion
    if(!localStorage.getItem("no-interface-banner")) {
```

Comment (acdha) [reply concordia/templates/transcriptions/asset_detail.html]: I haven't tested in awhile but wanted to confirm that this doesn't trigger a prompt by default in any of our supported browsers. It's possible that it'll fail for other reasons but [the code to detect that is a bit complex](https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#Testing_for_availability) and I think the failure mode in this case is probably better than trying something using cookies.

Comment (acdha) [reply concordia/templates/transcriptions/asset_detail.html]: I confirmed that this doesn't affect any of our supported browsers but confirming that on mobile reminded me that we might want to leave it hidden when “mobile” is in the user-agent and/or the screen size is below the 1024x768 threshold we're currently focused on:

https://github.com/LibraryOfCongress/concordia/blob/fc5cc5da4787280431592d5f46b7e81b5495baac/concordia/templates/action-app.html#L27

Comment (rstorey) [reply concordia/templates/transcriptions/asset_detail.html]: Done

**Pull request merged**